src/v0/destinations/tiktok_ads/networkHandler.js

```diff
@@ -1,24 +1,43 @@
 const { proxyRequest, prepareProxyRequest } = require('../../../adapters/network');
-const { processAxiosResponse } = require('../../../adapters/utils/networkUtils');
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
 const { DESTINATION } = require('./config');
-const { AbortedError, RetryableError } = require('../../util/errorTypes');
+const { NetworkError, ThrottledError, AbortedError } = require('../../util/errorTypes');
+const { TAG_NAMES } = require('../../util/tags');
+const { HTTP_STATUS_CODES } = require('../../util/constant');
 
 const responseHandler = (destinationResponse) => {
   const msg = `[${DESTINATION} Response Handler] - Request Processed Successfully`;
-  const { code } = destinationResponse.response;
-  const status = 200;
+  const {
+    response: { code },
+    status,
+  } = destinationResponse;
 
   if (code === 0 || code === 20001) {
     return {
-      status,
+      status: HTTP_STATUS_CODES.OK,
       message: msg,
       destinationResponse,
     };
   }
   if (code === 40100) {
-    throw new RetryableError(`Request failed with status: ${code}`, 429, destinationResponse);
+    throw new ThrottledError(`Request failed with status: ${code}`, destinationResponse);
   }
-  throw new AbortedError(`Request failed with status: ${code}`, 400, destinationResponse);
```

Comment:
Did we checked whether other destinations also need to be corrected ? To handle all the known scenarios, else pass status as-is.

Reply:
We have another task assigned for this. You can check it [here](https://www.notion.so/rudderstacks/Research-Network-Handlers-to-validate-if-responses-are-getting-handled-properly-21e7140185494c29876e17ee20a986e3?pvs=4)
We can take it up in future sprints